cli/medperf/config.py

```diff
@@ -1,11 +1,12 @@
 from os.path import expanduser, abspath
+from pathlib import Path
 
 version = "0.0.0"
 server = "https://api.medperf.org"
 certificate = None
 
 local_server = "https://localhost:8000"
-local_certificate = "server/cert.crt"
+local_certificate = str(Path(__file__).parent.parent.parent / "server/cert.crt")
```

Comment:
Why did this change have to be made? I'm worried that the fix might also fail for the same reason at some point. These kinds of bugs can be so annoying :)

Reply:
I see :)
I was looking again and I think this change is redundant. The error I faced seems to originate from another place.